common/views/components/ButtonOutlined/ButtonOutlined.tsx

```diff
@@ -10,8 +10,15 @@ import {
   ButtonIconWrapper,
 } from '../ButtonSolid/ButtonSolid';
 
-export const OutlinedButton = styled(BaseButton).attrs(props => ({
-  className: '',
+
+type MaybeAnchor = {
+  href?: string,
+}
+
+export const OutlinedButton = styled(BaseButton).attrs<MaybeAnchor>(props => ({
+  className: classNames({
+    'link-reset': !!props.href,
```

Comment:
Why would body text links not have an `href`?

Reply:
This styled component (`OutlinedButton`) can either be a button or an anchor, so we only want to give it a 'link-reset' class where necessary (when it's the latter). I suppose that it wouldn't be strictly necessary – adding the 'link-reset' class to it regardless wouldn't affect anything when it was a button, but feels better not to pepper the markup with classes that aren't going to do anything?